pkgs/top-level/all-packages.nix

```diff
@@ -17913,7 +17913,8 @@ julia_15 = callPackage ../development/compilers/julia/1.5.nix {
 
   iotop = callPackage ../os-specific/linux/iotop { };
 
-  iproute = callPackage ../os-specific/linux/iproute { };
+  iproute2 = callPackage ../os-specific/linux/iproute { };
+  iproute = iproute2; # Alias added 2020-11-15 (TODO: deprecate and move to pkgs/top-level/aliases.nix)
```

Comment:
Aliases belong into aliases.nix.

Reply:
Yes, the comment mentions this and it was intentionally not done yet (as for other attributes in all-packages.nix). Moving it there requires replacing all usages for CI and if/when I'll do that it'll be after a staging-next merge to minimize disruption.